src/anbox/container/management_api_stub.h

```diff
@@ -56,6 +56,7 @@ class ManagementApiStub : public DoNotCopyOrMove {
 
   mutable std::mutex mutex_;
   std::shared_ptr<rpc::Channel> channel_;
+  static const std::chrono::milliseconds stop_waiting_millis;
```

Comment:
This has to go into a anonymous namespace inside management_api_stub.cpp

Furthermore the variable type is already encoding that we have milliseconds here so no need to repeat that in the variable name. stop_waiting_timeout is what you want here as a name